datadog/resource_datadog_user.go

```diff
@@ -66,13 +84,26 @@ func resourceDatadogUserExists(d *schema.ResourceData, meta interface{}) (b bool
 	// Exists - This is called to verify a resource still exists. It is called prior to Read,
 	// and lowers the burden of Read to be able to assume the resource exists.
 	providerConf := meta.(*ProviderConfiguration)
-	client := providerConf.CommunityClient
 
-	if _, err := client.GetUser(d.Id()); err != nil {
-		if strings.Contains(err.Error(), "404 Not Found") {
-			return false, nil
+	if isV2User(d.Id()) {
```

Comment:
does this ID need to be the same as the V1 API. currently the v2 returns the uuid as the id and v1 returns the primary key as the id.

Reply:
Not sure I completely understand the question, but we store whichever ID the API returns, and make decision based on the format like here.